dotcom-rendering/src/lib/messagePicker.ts

```diff
@@ -1,4 +1,4 @@
-import { isUndefined, startPerformanceMeasure } from '@guardian/libs';
+import { startPerformanceMeasure } from '@guardian/libs';
```

Comment:
Can you explain the changes in this file? Ideally we would not change the messagePicker for this PR